Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
-ruby '2.7.5'
+ruby '>= 2.6.10'
 
-gem 'cocoapods', '~> 1.11', '>= 1.11.2'
+gem 'cocoapods', '~> 1.12'
```

Comment:
headsup, we're "bumping" cocoapods for everyone by doing this - are we sure we want it? is there a way we can be >= 1.11?